tickets/k8s/CoreV1Component.org

```diff
@@ -0,0 +1,196 @@
+# -*- ii: apisnoop; -*-
+#+TITLE: List Component Status
+#+AUTHOR: ii team
+#+TODO: TODO(t) NEXT(n) IN-PROGRESS(i) BLOCKED(b) | DONE(d)
+#+OPTIONS: toc:nil tags:nil todo:nil
+#+EXPORT_SELECT_TAGS: export
+* TODO Progress [2/5]                                                :export:
+- [X] APISnoop org-flow : [[https://github.com/cncf/apisnoop/blob/ComponentStatusSimpleTest/tickets/k8s/CoreV1Component.org][CoreV1ComponentStatus.org]]
+- [X] Blocked because of : [https://github.com/kubernetes/kubernetes/issues/86087]
```

Comment:
IT's not blocked per-se, it's not a test that is part of conformance.

Add a link to the https://github.com/cncf/apisnoop/blob/master/tickets/k8s/EndpointsToNotTest.org#componentstatus